pkg/config/config.go

```diff
@@ -347,7 +340,16 @@ func (f *File) Read(
 	errLog fsterr.LogInterface,
 	verbose bool,
 ) error {
-	var useStatic bool
+	// Ensure the static config is sound. This should never happen (tm).
+	// We are checking this earlier to simplify the code later on.
+	var staticConfig File
+	err := toml.Unmarshal(Static, &staticConfig)
+	if err != nil {
+		errLog.Add(err)
+		return invalidConfigErr(err)
```

Comment:
The error message `ErrInvalidConfig` is a bit vague for a user who doesn't know anything about the CLI's internal config or the static embed etc, we should update the error and make it clearer to the user what we're referring to and in this scenario it's actually quite unexpected.

Reply:
Sure. Do you have any suggestions?

Reply:
Updated.